src/main/java/net/minecraftforge/registries/GameData.java

```diff
@@ -575,6 +576,17 @@ public void onCreate(IForgeRegistryInternal<Item> owner, RegistryManager stage)
         }
     }
 
+    private static class EntityTypeCallbacks implements IForgeRegistry.BakeCallback<EntityType<?>>
+    {
+        static final EntityTypeCallbacks INSTANCE = new EntityTypeCallbacks();
+
+        @Override
+        public void onBake(IForgeRegistryInternal<EntityType<?>> owner, RegistryManager stage)
+        {
+            SpawnEggItem.finalizeModEggs();
```

Comment:
Why is this here and not in say `FMLCommonSetupEvent`?

Reply:
Because it seemed appropriate considering what the other bake callbacks are doing.
It makes even more sense with your concerns of who wins when registering the dispense behaviour.